src/Controller/FindByPid/GalleryController.php

```diff
@@ -3,27 +3,44 @@
 namespace App\Controller\FindByPid;
 
 use App\Controller\BaseController;
+use App\Ds2013\Presenters\Utilities\Paginator\PaginatorPresenter;
 use BBC\ProgrammesPagesService\Domain\Entity\Gallery;
 use BBC\ProgrammesPagesService\Domain\Entity\Image;
 use BBC\ProgrammesPagesService\Service\ImagesService;
+use BBC\ProgrammesPagesService\Service\ProgrammesAggregationService;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class GalleryController extends BaseController
 {
     public function __invoke(
         Gallery $gallery,
         ImagesService $imagesService,
-        ?string $imagePid
+        ?string $imagePid,
+        ProgrammesAggregationService $programmesAggregationService
+
     ) {
         $this->setIstatsProgsPageType('galleries_show');
         $this->setContextAndPreloadBranding($gallery);
+        $siblingLimit = 4;
 
         $images = $imagesService->findByGroup($gallery);
         $image = $this->getFirstImage($imagePid, $images);
+        $programme= $gallery->getParent();
```

Comment:
Add space after `$programme` or linter will fail.
```suggestion
        $programme = $gallery->getParent();
```